### PR TITLE
add Makefile for example

### PR DESCRIPTION
**Pull request by thinkerou**

### CODE REVIEW

File: examples/realtime-advanced/Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+deps:
```
Comment (appleboy) [inline examples/realtime-advanced/Makefile]: .PHONY: deps

File: examples/realtime-advanced/Makefile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	go get -d -v github.com/dustin/go-broadcast/...
+	go get -d -v github.com/manucorporat/stats/...
+
+build: deps
```
Comment (appleboy) [inline examples/realtime-advanced/Makefile]: .PHONY: build

Comment (thinkerou) [inline examples/realtime-advanced/Makefile]: I add `all`, so it have three now.

File: examples/realtime-chat/Makefile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+all: deps build
+
+deps:
+	go get -d -v github.com/dustin/go-broadcast/...
+
+build: deps
+	go build -o realtime-chat main.go rooms.go template.go
+
+.PHONY: \
```
Comment (appleboy) [inline examples/realtime-chat/Makefile]: I like the followings personally.

```
all: deps build
  
.PHONY: deps
deps:
    go get -d -v github.com/dustin/go-broadcast/...

.PHONY: build
build: deps
  	go build -o realtime-chat main.go rooms.go template.go
```

**Comment by thinkerou** [comment]
@appleboy Thanks, done.

**Comment by appleboy** [comment]
@thinkerou Thanks.

**Pull request merged**